lighthouse-core/test/lib/traces/tracing-processor-test.js

```diff
@@ -47,25 +47,6 @@ describe('TracingProcessor lib', () => {
     });
   });
 
-  it('doesn\'t throw when user_timing events have a colon', () => {
```

Comment:
ah true! :) can move this to `user-timing-test.js` to make sure we can inject a trace with these colons and everything goes through fine?